Development

dream-4ild opened the pull request.

Commit:
quite goos for the first time

Commit:
Add statistics

Commit:
fix bug with statistic

Commit:
add choosing file or language

Commit:
add default language

Commit:
add default language
pulled down the lines with the text a little bit

Commit:
add realtime statistics

Commit:
add main word. I suppose, this is quite good.

Commit:
add screeninfo

Commit:
add screeninfo

Commit:
try to fix screen width

Commit:
add space in the end of line

Commit:
update README.md, add space in the end of line and remove it from the last line of the text. properly organize methods in classes

Commit:
update README.md

polinavenberg commented on src/game.py:
```diff
@@ -0,0 +1,131 @@
+from time import sleep
+from tkinter import TclError
+from time import time
```
Неправильный порядок импортов, нужно поправить во всех файлах

dream-4ild replied:
поменял везде, оказывается, обычное ctrl+alt+L этого не делает, не знал, но ctrl+alt+O делает

polinavenberg commented on src/game.py:
```diff
@@ -0,0 +1,131 @@
+from src.constants import STATS_PATH, SECONDS_PER_MINUTE, STATS_UPDATE_FREQUENCY
+
+
+class game:
```
Название класса должно быть с большой буквы по правилам PEP8 (CamelCase)

dream-4ild replied:
Везде поменял

polinavenberg commented on src/game.py:
```diff
@@ -0,0 +1,131 @@
+            with open(STATS_PATH, 'w') as file:
+                file.write(f"{curr_score}\n")
+
+    def start(self):
```
Слишком большая функция (больше 40 строк), нужно разбить на несколько. Относится ко всем большим функциям в проекте

dream-4ild replied:
Эту и ещё две функции разбил на несколько

polinavenberg commented:
Огромный лайк тебе за твой гайд по запуску. Вот бы все такой писали

Commit:
change import order in every file.
 change class names to CamelStyle.
 Divide several methods into parts.
 Update README.md

dream-4ild commented:
Вроде всё исправил

Commit:
window.py

polinavenberg commented:
![image](https://github.com/dream-4ild/fast_typing_simulator/assets/79393802/7cf6831d-8f1e-4a6a-ac2e-dd44a15df9b2)
У меня слетела кодировка в русском языке. Я на винде запускала, у тебя все нормально работает?

polinavenberg commented:
Не получается знак вопроса ввести в "Are you?"

polinavenberg commented:
Хочется кнопку "Вернуться в главное меню" или что-то такое во время игры

polinavenberg commented:
Если проекты других людей меня не впечатлят, подниму оценку до 2-х
